docs/new/tutorials/near.md

```diff
@@ -0,0 +1,65 @@
+Tutorial on NEAR
+===================
+
+In this tutorial, you'll learn how to initialize a NEAR-based Substreams project using the Substreams CLI and the available NEAR development resources.
+
+{% hint style="info" %} 
+ The CLI installation is supported only on Linux and macOS. If you're using Windows, consider using the [DevContainer environment](../references/devcontainer-ref.md), which launches a Linux-based virtual environment.
+{% endhint %}
+
+## Step 1: Initialize Your NEAR Substreams Project
+
+1. [Install the Substreams CLI](../how-to-guides/cli/installing-the-cli.md).
+
+2. Currently, NEAR project templates are being developed for the `substreams init` command. In the meantime, you can get started with NEAR Substreams by cloning the existing example:
+
+   ```bash
+   git clone https://github.com/streamingfast/substreams-near-hello-world.git
+   cd substreams-near-hello-world
+   ```
+
+   This example demonstrates how to extract and process NEAR blockchain data using the [NEAR Full block](https://github.com/streamingfast/firehose-near/blob/develop/proto/sf/near/type/v1/type.proto) as input.
+
+{% hint style="info" %}
+**Coming Soon**: NEAR project templates will be available through `substreams init` in a future release, similar to other supported blockchains.
+{% endhint %}
```

Comment:
Change that to leverage `substreams init`, it will be ready once we publish it, no need to wait for an intermediate version.

> On it! I'll update the docs to leverage `substreams init` as requested. 🚀
> 
> 💻 [View my work](https://codegen.com/agent/trace/119962) • [🛑 Stop](https://codegen.com/agent/trace/119962/stop)